dev/tasks/README.md

```diff
@@ -166,14 +166,14 @@ Just render without applying or committing the changes:
 $ python crossbow.py submit --dry-run task_name
 ```
 
-Run only `conda` package builds but on all platforms:
+Run only `conda` package builds and a linux one:
 
 ```bash
-$ python crossbow.py submit conda-win conda-osx conda-linux
+$ python crossbow.py submit -g conda centos-7
```

Comment:
See the group definitions [here](https://github.com/kszucs/arrow/blob/b299c6ff515a33cffdabd0b974c12cdffcb97ebc/dev/tasks/tasks.yml#L18)